simul/monitor/bucket_stats_test.go

```diff
@@ -0,0 +1,68 @@
+package monitor
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
This is a rare occasion where true per method unit testing could've been done. 
However those two scenarios capture the essential.